chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -57,6 +57,10 @@ class ApiTestCaseWithTestReset(UtilityTestCase):
     _chroma_manager = None
     _unauthorized_chroma_manager = None
 
+    # Should any filesystems left running at the end of the test
+    # be torn down?
+    teardown_fs = True
```

Comment:
I'm not sure why we wouldn't want any FS to stay between tests.

Reply:
The way this test is written, it is needed because after the upgrade, the FS that was created before the upgrade is started up again to make sure all is working post-upgrade.

This used to be fine before we explicitly tore down filesystems in #299.

Now that we do that, this whole test probably needs to be rewritten to combine the individual tests into one single test.

Reply:
Fixed this.